barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -236,11 +253,14 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
     {
         auto width = full_honk_evaluations.size();
         std::vector<std::vector<FF>> first_level_coeffs(width >> 1, std::vector<FF>(2, 0));
-        for (size_t node = 0; node < width; node += 2) {
-            auto parent = node >> 1;
-            first_level_coeffs[parent][0] = full_honk_evaluations[node] + full_honk_evaluations[node + 1] * betas[0];
-            first_level_coeffs[parent][1] = full_honk_evaluations[node + 1] * deltas[0];
-        }
+        run_loop_in_parallel(width >> 1, [&](size_t start, size_t end) {
+            for (size_t node = start << 1; node < end << 1; node += 2) {
```

Comment:
should prolly clarify why it's `start << 1` rather than `start` or slightly rewrite